cr8/run_crate.py

```diff
@@ -214,6 +214,31 @@ def __enter__(self):
     def __exit__(self, *ex):
         self.stop()
 
+    def _obtain_http_url(self):
+        """
+        Read Crate log file to obtain HTTP address of node
+        """
+        def wait(waited, wait_time=0.25):
+            time.sleep(wait_time)
+            return waited + wait_time
+
+        time_waited = 0
+        logfile = os.path.join(self.logs_path, self.cluster_name + '.log')
+        while not os.path.exists(logfile):
```

Comment:
should there be a time_waited condition here as well in case the file is never created?